source/administration/config-settings.rst

```diff
@@ -5375,6 +5375,19 @@ In previous Mattermost Server versions, and this documentation, the instructions
 | This feature's ``config.json`` setting is ``"RunScheduler": true`` with options ``true`` and ``false``.                                 |
 +-----------------------------------------------------------------------------------------------------------------------------------------+
 
+Shared Channels (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Available in Enterprise Edition E20*
+
+**True**: Enables users from multiple Mattermost instances to collaborate with one another using shared channels.
+
+**False**: Disables channel sharing.
+
++---------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
The table is currently broken.

I believe the setting can also be documented in the telemetry doc.

Reply:
Thanks, @amyblais. I've fixed the table formatting, and I've confirmed that "bool EnableSharedChannels" is included in the Telemetry doc (though not via this PR).